### ISSUE TITLE
Some styles are not applied and I've got no ripple effect

### ISSUE THREAD

**Issue by ccjmne** [mdbootstrap/mdb-ui-kit#802]
Hi guys, I'm having some troubles...
I've created a clean empty project, with Bower, Gulp, Browserify and Sass. I wanna get into Bootstrap Material Design.

I am correctly importing all the required styles and fonts:

``` css
@import url('http://fonts.googleapis.com/css?family=Roboto:300,400,500,700');  
@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
@import 'bower_components/bootstrap/dist/css/bootstrap';
@import 'bower_components/bootstrap-material-design/dist/css/bootstrap-material-design.min';
@import 'bower_components/bootstrap-material-design/dist/css/ripples.min';
```

and all the JS files. I also use `$.material.init();`.

Yet, here's what I get:
![rendering](https://cloud.githubusercontent.com/assets/6822545/12124463/fb768064-b3e5-11e5-99ee-fd15d6fc87fc.png)

It is pretty neat, looks good for the most part. However, for example, you can notice that the styling for the `<div class="well">...</div>` is not right.

I've got some of the JS working, since the navbar is fully functioning. The form's animations are also fine for the most part:
- the inputs for email, text and password are **fine**
- the selects and multiselects are **fine**
- the checkboxes, radios and file input are **not properly styled**.

![rendering2](https://cloud.githubusercontent.com/assets/6822545/12124777/ab9a74b8-b3e7-11e5-832d-5262b791b774.png)

There's no problem whatsoever in the JSHint phase, nor in the browser's console. I've tried tweaking bits and pieces around but I can't get it right.

Any idea what I'm missing?

Checkout my project if you wanna have a look:

``` bash
git clone -b feature/bootstrap-material-design --single-branch https://github.com/ccjmne/first-aid-officers-maintenance-ui.git
npm install
bower install
npm start
```

**Comment by rosskevin** [comment]
We only work in html here, it seems you have two issues, 1 with ripples and 1 with radios.

Please create an isolated test case for each with the codepen requested here: https://github.com/FezVrasta/bootstrap-material-design/blob/master/CONTRIBUTING.md#bug-reports

We can then tell you what is going on from a structure perspected (or turn this into a bug).  

My guess without that?  1. you haven't included the ripples.js, 2. you have a different structure [than expected](http://fezvrasta.github.io/bootstrap-material-design/#radio-button).

A codepen will allow us to help you.

**Comment by rosskevin** [comment]
Also, as an FYI - v4 bootstrap has dropped wells in place of cards.  Also, wells aren't really "material", so if you are starting upfront, you might investigate a different container so it is more forward compatible.

**Comment by ccjmne** [comment]
Turns out that the only adjustment that had to be made was concerning my `$.material.init();` which was called... to early?
Wrapping it like follows:

``` javascript
$(function() {
    $.material.init();
});
```

... fixed **everything**.

I guess I'm just your typical back-end developer that gets stuck in each and every loophole of web development.
On the bright side, I learned that I should take a look at the "CONTRIBUTING.md" before asking questions. Sorry about that, won't happen again.

Yeah you're right about the wells.
I reckon I'll stick to Bootstrap3 here, since I'm not actually starting a project, just exploring material-compliant-ish Bootstrap extensions to swag up our product. I'll probably use Polymer on my next project if I get the chance.

Thanks very much!
